client/src/containers/edit/EditEducation.js

```diff
@@ -0,0 +1,114 @@
+import React, {Component} from 'react'
+import {
+ MenuItem,
+ TextField,
+ SelectField,
+ DatePicker,
+ RaisedButton
+} from 'material-ui'
+import Save from 'material-ui/svg-icons/content/save'
+import ContentAdd from 'material-ui/svg-icons/content/add'
+import {style} from "./styles";
+import './main.css'
+
+
+class EditEducation extends Component {
```

Comment:
These don't need to be components I don't think, unless you're thinking they will need to be in the future then ignore